demos/JWST/S1_miri_template.ecf

```diff
@@ -26,6 +26,16 @@ skip_gain_scale     False
 #Pipeline stages parameters
 jump_rejection_threshold  4.0 #float, default is 4.0, CR sigma rejection threshold
 
+#Background subtraction
+grouplevel_bg 		True
+ncpu				8
+bg_y1 				6
+bg_y2 				26
```

Comment:
I assume these are just a copy of what you used for NIRSpec? We'll need to choose values relevant to MIRI (and test whether they actually work for MIRI) before setting this step to run by default in the template